framework/framework.go

```diff
@@ -284,6 +294,36 @@ func ProcessDelete(ctx context.Context, obj interface{}, resources []Resource) e
 	return nil
 }
 
+// ProcessEvents takes the event channels created by the operatorkit informer
+// and executes the framework's event functions accordingly.
+func (f *Framework) ProcessEvents(ctx context.Context, deleteChan chan watch.Event, updateChan chan watch.Event, errChan chan error) {
```

Comment:
The new informer is more go idomatic because it works with channels. For our framework to be able to process the events we provide a convenience method here which can just be used within our operators.